### PR TITLE
Airport challenge update

### PR DESCRIPTION
**Pull request by FayeCarter**
Recap of week 1 learnings and application to new project

**Included in the change:**
- User stories completed
- Tests passing
- Have not worked on edge cases

**What is incomplete?**
- Edge cases not started
- After implementing stormy? method, I haven't had a chance to refactor the tests. I have a lot of repetition stormy or not stormy airport objects in tests
- Haven't removed magic number in tests

**What I'd like to improve:**
- How to improve use of context in tests
- Refactor tests to make them more readable

### CODE REVIEW

File: spec/airport_spec.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
+require 'airport'
+
+describe 'Airport' do
```
Comment (PhilipVigus) [inline spec/airport_spec.rb]: Your general layout of this file is excellent. I like the use of contexts and there is really good consistency with the syntax you've used in the lets

File: spec/airport_spec.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+  subject(:airport) { Airport.new }
```
Comment (PhilipVigus) [inline spec/airport_spec.rb]: I'm not understanding why you've used a let and a subject for essentially the same thing. My understanding is that a let is generally preferable in this situation, and would suggest using it for both.

File: spec/airport_spec.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  let(:plane) { Plane.new }
+
+  context 'Airport can #land planes' do
+
+    it 'Airport responds to #land' do
+      expect(airport).to respond_to(:land)
+    end
+
+    it 'Plane lands in airport' do
+      allow(airport).to receive(:stormy?).and_return false
```
Comment (PhilipVigus) [inline spec/airport_spec.rb]: The repeated use of this line across multiple tests could be extracted out into a let. This would make your code DRY and much easier to read.

File: spec/airport_spec.rb [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    it "Plane  won't land if airport is full" do
+      allow(airport).to receive(:stormy?).and_return false
+      airport.land
+      expect { airport.land }.to raise_error 'Airport Full'
+    end
+
+  end
+
+  context 'Airport can #take_off planes' do
```
Comment (PhilipVigus) [inline spec/airport_spec.rb]: I'm curious about your use of # here as I haven't seen it before. I'd be grateful if you could explain what it means when you use it in this way?

File: spec/airport_spec.rb (hunk 2 above)
Comment (PhilipVigus) [inline spec/airport_spec.rb]: As far as I'm aware, you could remove this by tweaking your describe on line 3. If you replace describe 'Airport' with describe Airport, then subject will automatically resolve to an instance of the Airport class.

**Comment by PhilipVigus** [comment]
Generally really impressed with your code. Most of it is really laid out well, although there are some things you can do in the test files to improve readability a bit.
